common/app/routes/Challenges/views/quiz/Quiz.jsx

```diff
@@ -62,13 +62,13 @@ const propTypes = {
   correct: PropTypes.number,
   currentIndex: PropTypes.number,
   dashedName: PropTypes.string,
-  description: PropTypes.string,
+  description: PropTypes.array,
```

Comment:
This should really be `PropTypes.arrayOf(PropTypes.string)`

Reply:
Hi @Bouncey ! 
The description object is actually is an array of objects and not strings.
![screenshot from 2018-03-07 17-51-39](https://user-images.githubusercontent.com/21025052/37092176-c74c7062-2230-11e8-9e4f-7cb63f638026.png)

So, should i change it to PropTypes.arrayOf(PropTypes.object) ? or something else?

Reply:
```js
const propTypes = {
  // ...
  description: PropTypes.arrayOf(
    PropTypes.shape({
      answer: PropTypes.number,
      choices: PropTypes.arrayOf(PropTypes.string),
      explanation: PropTypes.string
      question: PropTypes.string
      subtitle: PropTypes.string
    })
  ),
  // ...
};
```